client/src/components/NewUserWelcome/NewUserWelcome.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+    <div>
+        <ConfigProvider v-slot="config">
+            <component
+                :is="viewElement"
+                v-bind="currentNode"
+                @select="down"
+                @back="up"
+                :image-loc="config.welcome_directory"
+            >
+            </component>
+        </ConfigProvider>
+    </div>
+</template>
+<script>
+import { BCard, BCardGroup, BTabs, BTab, BCarousel, BCarouselSlide, BButton, BRow, BCol } from "bootstrap-vue";
+import { getAppRoot } from "onload/loadConfig";
+import Topics from "components/NewUserWelcome/components/Topics";
+import Subtopics from "components/NewUserWelcome/components/Subtopics";
+import Slides from "components/NewUserWelcome/components/Slides";
+import ConfigProvider from "components/providers/ConfigProvider";
+
+export default {
+    components: {
+        BCard,
+        BCardGroup,
+        BTabs,
+        BTab,
+        BCarousel,
+        BCarouselSlide,
+        BButton,
+        BRow,
+        BCol,
+        Topics,
+        Subtopics,
+        Slides,
+        ConfigProvider,
+    },
+    data() {
+        return {
+            position: [],
+        };
+    },
+    props: {
+        newUserDict: { type: Object, required: true },
+    },
+    computed: {
+        depth() {
+            return this.position.length;
+        },
+        currentNode() {
+            return this.position.reduce((node, i) => {
+                return node.topics[i];
+            }, this.newUserDict);
+        },
+        viewElement() {
+            let element;
+            switch (this.depth) {
+                case 0:
+                    element = Topics;
+                    break;
+                case 1:
+                    element = Subtopics;
+                    break;
+                case 2:
+                    element = Slides;
+                    break;
+            }
+            return element;
+        },
+    },
+    methods: {
+        imgUrl(src) {
+            const root = getAppRoot();
+            const url = `${root}static/${this.imageLoc}${src}`.replace("//", "/");
+            return url;
+        },
+        up() {
+            this.position.pop();
+        },
+        down(index) {
+            this.position.push(index);
+        },
+    },
+};
+</script>
+
+<style type="text/css">
```

Comment:
These custom styles conflict with theme settings. I think we need to remove them.